database/common.go

```diff
@@ -0,0 +1,43 @@
+package database
+
+import (
+	"database/sql"
+	"fmt"
+	"github.com/gmo-personal/coding_challenge/server/utils"
+	_ "github.com/go-sql-driver/mysql"
+	"os"
+)
+
+var db *sql.DB
+
+// Inits the database with schema name teleport and all tables
+func InitDatabase() {
```

Comment:
avoid using global vars
return `sql.DB `from `InitDatabase` and have the caller pass it around
or better - make a wrapper type that has methods for all the DB operations

Reply:
will fix